_includes/head.html

```diff
@@ -1,6 +1,6 @@
 <head>
   <meta charset="utf-8"/>
-  <meta http-equiv="X-UA-Compatible" content="IE=edge"//>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
```

Comment:
good catch 👍

Reply:
That wasn't me, that was https://github.com/prettier/prettier

Reply:
I like it when robots fix things :)